hypothesis-python/tests/cover/test_float_nastiness.py

```diff
@@ -186,15 +186,16 @@ def test_floats_in_tiny_interval_within_bounds(data, center):
     lo = Decimal.from_float(next_down(center)).next_plus()
     hi = Decimal.from_float(next_up(center)).next_minus()
     assert float(lo) < lo < center < hi < float(hi)
-    val = data.draw(st.floats(lo, hi))
-    assert lo < val < hi
+    with pytest.raises(HypothesisDeprecationWarning):
+        val = data.draw(st.floats(lo, hi))
+        assert lo < val < hi
 
 
 def test_float_free_interval_is_invalid():
     lo = (2 ** 54) + 1
     hi = lo + 2
     assert float(lo) < lo < hi < float(hi), 'There are no floats in [lo .. hi]'
-    with pytest.raises(InvalidArgument):
+    with pytest.raises((InvalidArgument, HypothesisDeprecationWarning)):
```

Comment:
Instead of editing this test, just add the `@checks_deprecated_behaviour` decorator (and same for the others that fail because of the new deprecation).